_i18n/en.yml

```diff
@@ -801,7 +801,24 @@ roadmap:
   p2pool: P2Pool released
   haveno: Haveno
 
-
+  ringsize-sera-tryp: increase ring size to 100+ through Seraphis or Triptych
+  seraphis-jamtis: Seraphis & jamtis
+  ospead: OSPEAD
+  released-0-17-3-0-cli: CLI 0.17.3.0 released
+  released-0-17-3-0-gui: GUI 0.17.3.0 released
+  released-0-17-3-1-gui: GUI 0.17.3.1 released
+  released-0-17-3-2-gui: GUI 0.17.3.2 released
+  released-0-17-3-2-cli: CLI 0.17.3.2 released
+  released-0-18: CLI and GUI 0.18 "Flourine Fermi" released
+  hardfork-v15: Network upgrade v15 Hardfork
+  ringsize-11-16: Ringsize increase 11 -> 16
+  viewtags: Viewtags (upto -40% wallet sync time)
+  feechanges: Fee changes
```

Comment:
This is too generic.

Reply:
In the spirit of less clutter, i decided to just keep the link to the v15 network upgrade blog (which contains all the highlights) let me know if this is ok, or if we should specifically list some highlights in the roadmap.

Reply:
I would add some details or remove it. You have the link to the blog above that contains all the details in any case.